app/templating/summary/question.py

```diff
@@ -1,63 +1,82 @@
 import collections
 
+from app.forms.household_relationship_form import build_relationship_choices
 from app.templating.summary.answer import Answer
 from app.templating.utils import get_question_title
 
 
 class Question:
 
-    def __init__(self, question_schema, answer_store, metadata, schema):
-        self.id = question_schema['id']
+    def __init__(self, question_schema, answer_store, metadata, schema, group_instance):
+        self.id = question_schema['id'] + '-' + str(group_instance)
         self.type = question_schema['type']
-
+        self.schema = schema
         answer_schemas = iter(question_schema['answers'])
 
         # Using group instance as 0 for now as summary rendering context only has knowledge of current locations group instance (i.e. 0)
-        self.title = get_question_title(question_schema, answer_store, schema, metadata, group_instance=0) or question_schema['answers'][0]['label']
+        self.title = (get_question_title(question_schema, answer_store, schema, metadata, group_instance=group_instance) or
+                      question_schema['answers'][0]['label'])
         self.number = question_schema.get('number', None)
-        self.answers = self._build_answers(answer_store, question_schema, answer_schemas)
+        self.answers = self._build_answers(answer_store, question_schema, answer_schemas, group_instance)
 
     @staticmethod
-    def _get_answers(answer_store, answer_id):
-        return answer_store.filter(answer_ids=[answer_id]).escaped().values()
+    def _get_answers(answer_store, answer_id, group_instance):
+        answers = answer_store.filter(answer_ids=[answer_id], group_instance=group_instance).values()
```

Comment:
Why don't we escape these anymore?

Reply:
I'm not sure...

Tests still pass with `escaped()` . I can't see why it was removed. @jonnyshaw89 Any ideas?